app/models/taging.rb

```diff
@@ -0,0 +1,13 @@
+class Taging < ActiveRecord::Base
```

Comment:
@rakesh-1010 The model name should be `Tag` as it contains list of tags for tasks.